gef.py

```diff
@@ -3483,8 +3483,13 @@ def hook_stop_handler(_: "gdb.StopEvent") -> None:
 def new_objfile_handler(evt: Optional["gdb.NewObjFileEvent"]) -> None:
     """GDB event handler for new object file cases."""
     reset_all_caches()
+    path = evt.new_objfile.filename if evt else gdb.current_progspace().filename
     try:
-        target = pathlib.Path( evt.new_objfile.filename if evt else gdb.current_progspace().filename)
+        if gef.session.root:
+            # If the process is in a container, replace the 'target:' placeholder
+            # with the actual root directory of the process.
+            path = path.replace('target:', str(gef.session.root))
```

Comment:
We use double quotes, and let's use `ltrim` so that we don't accidentally remove it from anywhere else (even though a colon shouldn't appear)

Reply:
I opted for `str.startswith` + `str.replace('target:', .., 1)` to make sure we only ever replace a *prefix*